GetIntoTeachingApiTests/Controllers/SchoolsExperience/CandidatesControllerTests.cs

```diff
@@ -119,15 +125,34 @@ public void SignUp_ValidRequest_UpsertsCandidateAndRespondsWithSuccess()
         }
 
         [Fact]
-        public void SignUp_WhenCrmIntegrationIsPaused_RaisesError()
+        public void SignUp_WhenCrmIntegrationPaused_QueuesCandidateUpsert()
         {
+            var request = new SchoolsExperienceSignUp { FirstName = "first" };
             var mockAppSettings = new Mock<IAppSettings>();
             mockAppSettings.Setup(m => m.IsCrmIntegrationPaused).Returns(true);
+
+            var response = _controller.SignUp(request, mockAppSettings.Object);
+
+            var created = response.Should().BeOfType<CreatedAtActionResult>().Subject;
+            var signUp = created.Value.Should().BeAssignableTo<SchoolsExperienceSignUp>().Subject;
+            signUp.FirstName.Should().Be(request.FirstName);
```

Comment:
Worth asserting that the `signUp.Id` is not null here? may also be worth adding a quick test around the production behaviour and/or a comment to say that its temporary while we roll it out